client/lib/url/test/is-external.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/**
+ * Internal dependencies
+ */
+import isExternal from '../is-external';
+
+describe( 'isExternal', () => {
+	test( 'should return false for relative path-only url', () => {
+		const source = '/relative';
+
+		const actual = isExternal( source );
+
+		expect( actual ).toBeFalse;
```

Comment:
These look like chai expects, is this actually testing anything?

We can either rewrite them all for jest expect (preferred: `.toBe( false )`), or pull the import from the file they came from:

```js
import { expect } from 'chai';
```